allennlp/interpret/saliency_interpreters/integrated_gradient.py

```diff
@@ -0,0 +1,98 @@
+# pylint: disable=protected-access
+import math
+from typing import List, Dict
+import numpy
+from allennlp.common.util import JsonDict, sanitize
+from allennlp.interpret.saliency_interpreters import SaliencyInterpreter
+from allennlp.modules.text_field_embedders import TextFieldEmbedder
+from allennlp.data import Instance
+
+@SaliencyInterpreter.register('integrated-gradients-interpreter')
```

Comment:
I wouldn't include `-interpreter` in the registered name - that part will always be obvious when this string is used.  Same for the other ones.